lib/loader/fs-resolver.js

```diff
@@ -90,7 +99,7 @@ FSResolver.prototype = {
       if (base) {
         local = path.relative(base, local);
       }
-      if (root) {
+      if (root && !isAbsolute(local)) {
```

Comment:
I'm OK merging as-is, then. Could you add a comment with a TODO to remove it once 0.10 support is no longer needed?